pkg/util/scrubber/scrubber_test.go

```diff
@@ -83,3 +83,13 @@ func TestScrubLine(t *testing.T) {
 	res := scrubber.ScrubLine("https://foo:bar@example.com")
 	require.Equal(t, "https://foo:********@example.com", res)
 }
+
+func TestGitleaks(t *testing.T) {
+	scrubber := New()
+	scrubber.gitleaks([]byte(`hello
```

Comment:
I think we may need an additional test to invoke `scrubber.scrubReader` directly or indirectly. While the existing test can check that gitleaks works as expected it won't ensure that `gitleaks` filtering is in the relevant scrubbing path.